server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -2375,15 +2383,22 @@ public JobDetail exportDataAsync(
         @QueryParam("ext")
         @ApiParam(value = "Key/Value pairs to be passed to the extension adapter when generating a manifest",
         required = false, example = "ext=version:1.2.3&ext=extension_key:EXT1")
-        List<KeyValueParameter> extensionArgs) {
+        List<KeyValueParameter> extensionArgs) throws JobException {
 
         Consumer consumer = consumerCurator.verifyAndLookupConsumer(consumerUuid);
         ConsumerType ctype = this.consumerTypeCurator.getConsumerType(consumer);
 
+        Owner test = consumer.getOwner();
+
         Owner owner = ownerCurator.findOwnerById(consumer.getOwnerId());
 
-        return manifestManager.generateManifestAsync(consumerUuid, owner, cdnLabel, webAppPrefix,
-            apiUrl, getExtensionParamMap(extensionArgs));
+        log.info("TEST: {} => {}", test, owner);
```

Comment:
Better to remove this (and the added line above) since it is accidentally leftover test logging.

Reply:
Done